Library/Formula/tmux.rb

```diff
@@ -43,14 +43,26 @@ def install
 
     system "make", "install"
 
-    bash_completion.install "examples/bash_completion_tmux.sh" => "tmux"
-    pkgshare.install "examples"
+    if head?
```

Comment:
@DomT4 - `build.head?` was actually nil in `caveats`. I saw this issue - https://github.com/Homebrew/homebrew/issues/37120 and using `head?` solved it for me.

I can switch it back in the `install` method, but it won't work in `caveats`.

Thoughts? Happy to switch to whatever is best